ClassFieldTheory/GroupCohomology/IndCoind/TrivialCohomology.lean

```diff
@@ -0,0 +1,259 @@
+/-
```

Comment:
What is this file? This should not be in this PR?

Reply:
Indeed!